src/Export/Adapters/PriceAdapter.php

```diff
@@ -33,7 +56,10 @@ public function __construct(
      */
     public function adapt(ProductEntity $product): array
     {
-        $prices = $this->getPricesFromProduct($product);
+        $prices = (!Utils::versionLowerThan('6.4.9.0') && $this->config->getAdvancedPricing() !== AdvancedPricing::OFF)
```

Comment:
This check needs to look like this:
https://github.com/findologic/plugin-shopware-6/blob/5d3b96a79b62a3f0f5a114b5e300081ea2acee5c/src/Core/Content/Product/SearchKeyword/ProductSearchBuilder.php#L54

And please also create a `useAdvancedPricing()` function for this check

Reply:
And please use `versionGreaterOrEqual`

Reply:
- I meant to move the whole check to the function, not only the version check
- The current brackets around the check are unnecessary